libs/common/src/types/mongo-db-models/cron-job-logs.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This type is not well defined in external-api,
+ * but causes type errors in external-api where
+ * `eStore` related files use the db client
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type CronJobLog = any;
```

Comment:
This type is a bit of a hack to fix some `tsc --noemit` errors in `external-api`

Reply:
This might affect an eStore API PR in review - so just mentioning when we merge main into SQL we might need to revisit. Happy to keep just like this for now though :) explanation makes sense!